incubator/scylladb/README.md

```diff
@@ -0,0 +1,140 @@
+# ScyllaDB Helm Chart on Google Kubernetes Engine (GKE)
+
+ ## What is Helm Charts? Why use a Helm Chart? 
+  
+Helm uses a packaging format called charts. A chart is a collection of files that describe a related set of Kubernetes resources. A single chart might be used to deploy something simple, like a memcached pod, or something complex, like a full web app stack with HTTP servers, databases, caches, and so on. 
+
+Helm Charts deploys all the kubernetes entities in a ordered fashion wrapping them together in a RELEASE. In addition to that you also get versioning control allowing you to upgrade your release and rolling back changes.
+
+A good introduction to Helm Charts by Amy Chen can be found [here](https://youtu.be/vQX5nokoqrQ)
+
+## Running on GKE:
```

Comment:
IMO the installation instructions should be as generic as possible. You could take a look at the other chart docs for reference

Reply:
I understand what you are saying and it makes a lot of sense. But I also wanted to give an opportunity to current users of the product to use this chart as their first k8s/helm experience and thought it would be nice to have some instructions for newcomers. If you think it is an absolute must to drop all the helm instructions, I will.